vtStorAtaProtocol/Platform/Linux/ProtocolAtaPassThrough.h

```diff
@@ -30,33 +31,74 @@ namespace vtStor
 namespace Protocol
 {
 
+const U32 SG_IO_INTERFACE_ID             = 'S';
+const U8  SG_ATA_PASS_THROUGH_16         = 0x85;
+const U8  SG_CDB_16_LEN                  = 16;
+const U8  MAX_SB_LEN                     = 32;
+
+const U8  SG_ATA_LBA48                   = 0x01;
+const U8  SG_ATA_PROTO_NON_DATA          = 0x06;
+const U8  SG_ATA_PROTO_PIO_IN            = 0x08;
+const U8  SG_ATA_PROTO_PIO_OUT           = 0x0A;
+const U8  SG_ATA_PROTO_DMA               = 0x0C;

```

Comment:
I thinks we should not change to Bit Fields, beacause it need simple. Changing to Bit Fields is not help us understand format. Ex: SG_ATA_PROTO_NON_DATA = 0x06, it's same 1 << 2 | 1 << 1 or 3 << 1. I researched documents. It's same our code.